solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/util/XMLMessageMapper.java

```diff
@@ -207,6 +214,11 @@ public XMLMessage map(Message<?> message, Collection<String> excludedHeaders, bo
 				.setHeaderIfAbsent(MessageHeaders.CONTENT_TYPE, xmlMessage.getHTTPContentType())
 				.setHeaderIfAbsent(IntegrationMessageHeaderAccessor.DELIVERY_ATTEMPT, new AtomicInteger(0));
 
+		if (isNullPayload) {
+			logger.info("Null payload detected, setting Spring header " + SolaceBinderHeaders.NULL_PAYLOAD);
```

Comment:
Remove this log or change it to a DEBUG log (wrap it in a `logger.isDebugEnabled()`), or a TRACE log (wrap it in a `logger.isTraceEnabled()`). Logging on each message will bog down performance.

Reply:
Done